src/index.js

```diff
@@ -64,14 +71,41 @@ export function filterCards(cards, options) {
 	return cards.filter(card => !card.back.some(token => token.content.trim().includes('<!-- md2anki ignore-card -->'.trim())));
 }
 
-export function deckFromCards(cards, options) {
+export function deckFromCards(cards, images, options) {
 	// create new deck
 	const apkg = new AnkiDeck(options.deckName, { css: '' });
 	console.log(`deck initialized!`);
+	// add media files to deck
+	images.forEach(image => {
+		apkg.addMedia(image, fs.readFileSync(image));
+	});
 	// add cards to deck (convert tokens to html)
-	cards.forEach((card, i) => {
+	cards.forEach(card => {
 		apkg.addCard(md.renderer.render(card.front, md.options, {}), md.renderer.render(card.back, md.options, {}));
 	});
 	console.log(`added ${cards.length} cards to the deck!`);
+	console.log(`added ${images.length} images to the deck!`);
 	return apkg;
+}
+
+export function imagesFromTokens(tokens, inputPath) {
+	let images = [];
+	tokens.forEach((token) => {
+		if (token.type === 'image') {
+			// find images in tokens and save them to a list
+			let filePath = path.join(path.dirname(inputPath), token.attrGet('src'));
+			filePath = filePath.split(path.sep).join(path.posix.sep);
+			images.push(filePath);
+		}
+		if (token.type === 'inline') {
+
+			// recursively find images in children
+			images.push(imagesFromTokens(token.children, inputPath));
+		}
+	});
+
+	// flatten the list to a depth of 1
+	images = images.flat();
+
+	return images.filter(image => image.length > 0);
```

Comment:
```suggestion
	images = images.filter(image => image.length > 0);
	return [...new Set(images)];
```

Bilder sollten nicht doppelt eingebunden werden